samples/shell/src/main.c

```diff
@@ -34,9 +34,6 @@ void main(void)
 
 	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
```

Comment:
Can be deleted as well?

Reply:
Removed

Reply:
Restored as previously branch was based on main, not on develop branch. Now this line cannot be removed, npmx_instance is used below.